template/cordova/lib/MSBuildTools.js

```diff
@@ -46,11 +46,8 @@ MSBuildTools.prototype.buildProject = function(projFile, buildType, buildarch, o
     '/p:Configuration=' + buildType,
     '/p:Platform=' + buildarch];
 
-    if (otherConfigProperties) {
-        var keys = Object.keys(otherConfigProperties);
-        keys.forEach(function(key) {
-            args.push('/p:' + key + '=' + otherConfigProperties[key]);
```

Comment:
The former one. Please see [build.js:344](https://github.com/apache/cordova-windows/pull/217/files/6308e6974f5499857254fb79c91ded422b46486f#diff-9d172d6844120225b6cfb763491e4e03R344)